Explorer/Assets/DCL/AvatarRendering/Wearables/Helpers/IWearableAssetsCache.cs

```diff
@@ -11,21 +12,20 @@ public enum ReleaseResult
             /// </summary>
             ReturnedToPool,
 
-            /// <summary>
-            ///     Indicates that the asset was not returned to the pool because the pool is full
-            ///     so the asset should be destroyed.
-            ///     It is the responsibility of the caller to destroy the asset
-            /// </summary>
-            CapacityExceeded,
-
             /// <summary>
             ///     Indicates the the environment is being disposed so no actions related to the asset should be performed
             /// </summary>
             EnvironmentIsDisposing,
         }
 
-        bool TryGet(GameObject asset, out GameObject instance);
+        List<CachedWearable> AllCachedWearables { get; }
+
+        Dictionary<WearableAsset, List<CachedWearable>> Cache { get; }
```

Comment:
Please check if we ever need to expose the dictionary. If so expose it as `IReadOnly`

Reply:
I see you use it: `ProfilingCounters.WearablesAssetsInCacheAmount.Value = wearableAssetsCache.Cache.Keys.Count;`. Just expose Count instead. And not `Keys.Count`, simply `Count`

Reply:
removed as well `AllCachedWearables` , since it was never used